day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -224,6 +230,8 @@ the `/var/lib/origin/openshift.local.etcd` directory.
 ====
 Because clusters upgraded from previous versions of {product-title} might
 contain v2 data stores, back up both v2 and v3 datastores.
+
+If you do not have a v2 datastore, you can back up only the v3 datastore.
```

Comment:
This sentence seems a bit odd, or obvious, to me. Maybe something like:
_Because clusters upgraded from previous versions of {product-title} might
contain v2 data stores, back up all etcd datastores, including any v2 and v3 datastores._

Reply:
Thanks!